assets/components/errorMessage/errorMessage.scss

```diff
@@ -20,18 +20,19 @@
     width: 404px;
   }
 
-  svg {
+  .svg-exclamation {
```

Comment:
Is it possible to remove the nesting here? It is possible to do it by using a BEM selector like `component-error-message__svg-exclamation`.

Reply:
Not without rewriting our SVG component. We can do that in a separate PR.

Reply:
👍